include/celero/Print.h

```diff
@@ -41,6 +41,7 @@ namespace celero
 		void Failure(const std::string& x);
 		void Run(std::shared_ptr<Result> x);
 		void Done(std::shared_ptr<Result> x);
+		void DoneExperiment(std::shared_ptr<Experiment> x);
```

Comment:
Agreed.  Can you think of a different way to output this information while making it more clear as to what the individual values mean?  I am not opposed to your change in general, I just recognize that some the output is ambiguous and don't want to compound the problem.